storage-node/src/commands/server.ts

```diff
@@ -152,10 +155,13 @@ Supported values: warn, error, debug, info. Default:debug`,
     }
 
     const storageProviderAccount = this.getAccount(flags)
+    const workerId = flags.worker
+    if (!(await verifyWorkerId(api, flags.queryNodeEndpoint, workerId, storageProviderAccount.address))) {
+      this.exit(ExitCodes.InvalidWorkerId)
```

Comment:
It might be more convenient for the storage node operator if instead of exiting, we can loop trying to recheck valid keys and worker id. It allows the node operator to start their node before being assigned buckets. And as soon as they are assigned buckets and accept invite the server api will come up.

Reply:
It is possible now to start the node without any buckets assigned, this will only emit a warning: https://github.com/Joystream/joystream/pull/4283/files#diff-c2831ec0c824e9dbc4d17c21757e3ec83fc0a97495066219cb69f350414dbb9eR288